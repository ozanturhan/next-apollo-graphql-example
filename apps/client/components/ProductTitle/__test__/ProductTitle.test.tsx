import React from 'react';
import 'jest-styled-components';
import { render } from '@testing-library/react';
import { ProductTitle } from '../ProductTitle';

it('renders correctly', () => {
  const tree = render(<ProductTitle>Hello Product</ProductTitle>).container;
  expect(tree).toMatchSnapshot();
});
