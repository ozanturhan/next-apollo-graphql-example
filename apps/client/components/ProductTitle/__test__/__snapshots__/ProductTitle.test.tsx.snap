// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
.c0 {
  font-weight: bold;
  color: #4d4d4d;
  font-size: 1rem;
  overflow: hidden;
  text-overflow: ellipsis;
  display: -webkit-box;
  -webkit-line-clamp: 2;
  -webkit-box-orient: vertical;
  margin-bottom: 15px;
  width: calc(100% - 10px);
}

<div>
  <span
    class="c0"
  >
    Hello Product
  </span>
</div>
`;
